example-proxy.ini

```diff
@@ -20,8 +20,6 @@ instance = http://192.0.2.2
 # point to a directory containing trusted roots (only .pem files are considered!)
 #endpoint = tls:host=foo:port=636:trustRoots=/path/to/pems
 endpoint = tcp:host=192.0.2.1:port=389
-# Enabling/disabling of STARTTLS (currently unsupported)
-use-tls = false
```

Comment:
Why would you remove this from the example config?
I would keep it as documentation like

    # Enable /disbale STARTTLS, default=false
    #use-tls = true